common/lib/xmodule/xmodule/js/src/sequence/edit.coffee

```diff
@@ -1,2 +1 @@
 class @SequenceDescriptor extends XModule.Descriptor
-
```

Comment:
Looks like this file initially had two blank lines at the end of the file, now it just has one. Here's my editor with this file open on `master`:

![](https://i.bjacobel.com/20170104-7t7ob.png)